scripts/generate_16s_18s_its_primer_sql

```diff
@@ -1644,8 +2416,9 @@ def generate_insert_values(last_plate_id, last_container_id,
 
 
 if __name__ == "__main__":
+    # use last_plate_id=10, last_container_id=1536,  for 18S + ITS generation
```

Comment:
Does this mean that we are not adding 18S + ITS or that we are adding 16S + 18S + ITS? It's confusing looking at the changes and more if we leave this comment there ...